core/config/general_config.go

```diff
@@ -591,6 +592,12 @@ func (c *generalConfig) Dev() bool {
 	return c.viper.GetBool(envvar.Name("Dev"))
 }
 
+// ShutdownGracePeriod is the maximum duration of graceful application shutdown.
+// If exceeded, it will try closing DB lock and connection and exit immediately to avoid SIGKILL.
+func (c *generalConfig) ShutdownGracePeriod() time.Duration {
+	return c.getWithFallback("ShutdownGracePeriod", parse.Duration).(time.Duration)
+}
+
```

Comment:
Don't forget to add it to the docs here: https://github.com/smartcontractkit/documentation/pull/604/files

Reply:
https://github.com/smartcontractkit/documentation/pull/604/commits/5575fe59ef93e91452d854da073a96e6893680bb